packages/account/src/Components/api-token/__tests__/api-token-clipboard.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { screen, render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ApiTokenClipboard from '../api-token-clipboard';
+
+const modal_root_el = document.createElement('div');
+modal_root_el.setAttribute('id', 'modal_root');
```

Comment:
Please move this assignments to `describe` suite and add cleanup functions to remove the mock modal element